nUpdate.Shared/Updating/UpdateManager.cs

```diff
@@ -223,6 +223,11 @@ public CultureInfo LanguageCulture
         /// </remarks>
         public bool UseCustomInstallerUserInterface { get; set; }
 
+        /// <summary>
+        ///     Gets or sets a value indicating the timeout of search for updates.json. In milliseconds. Default 10000.
```

Comment:
"Gets or sets a value indicating" should be used when documenting boolean properties. In this case, I'd prefer "Gets or sets the timeout that should be used when searching for updates. By default, this is set to 10000 milliseconds."